### ISSUE TITLE
Aggregration-Functions

### ISSUE THREAD

**Issue by Disane87** [mevdschee/php-crud-api#60]
It would be nice, if there is a feature to count or sum up all entries in a table, something like that:
`api?Aromen&aggregate=count&filter=USER_ID,eq,1`

If I want to sum up a table I have to choose a col to sum:
`api?Aromen&aggregate=sum&col=price&filter=USER_ID,eq,1`

It's only a thought to enhance this great API.

**Comment by mevdschee** [comment]
Good idea, but I'm not sure whether or not this should be included or solved by:

https://github.com/mevdschee/php-sp-api

If we are going to be fully expressive on aggregate functions we need group as an argument.

Not sure whether or not this will open up the api to allow expensive queries (something I try to avoid).

**Comment by jahd2602** [comment]
I find this repository to be a great tool for RAD, and adding aggregation functions would make it a lot greater. Would it be possible to implement it in a database agnostic way? Or should it be implemented for every database supported?
